archivist/subjects.py

```diff
@@ -91,20 +92,46 @@ def create(
             ),
         )
 
-    def import_subject(self, name: str, subject: Subject) -> Subject:
+    def share(
+        self, name: str, other_name: str, other_archivist: archivist.Archivist
+    ) -> Tuple[Subject, Subject]:
```

Comment:
There is no need to import the tuple type

``` python
from __future___ import annotations

def share(
        self, name: str, other_name: str, other_archivist: archivist.Archivist
    ) -> (Subject, Subject):
````

Reply:
task: [check] ./scripts/builder.sh python3 -m pyright archivist
No configuration file found.
pyproject.toml file found at /home/builder.
Loading pyproject.toml file at /home/builder/pyproject.toml
Assuming Python version 3.7
Assuming Python platform Linux
Auto-excluding **/node_modules
Auto-excluding **/__pycache__
Auto-excluding **/.*
Searching for source files
Found 48 source files
pyright 1.1.272
/home/builder/archivist/subjects.py
  /home/builder/archivist/subjects.py:98:10 - error: Tuple expression not allowed in type annotation
    Use Tuple[T1, ..., Tn] to indicate a tuple type or Union[T1, T2] to indicate a union type
  /home/builder/archivist/subjects.py:98:10 - error: Expected class type but received "tuple[Type[Subject], Type[Subject]]" (reportGeneralTypeIssues)
2 errors, 0 warnings, 0 informations 
Completed in 1.571sec